packages/client-sdk-web/src/internal/pubsub-client.ts

```diff
@@ -134,6 +134,13 @@ export class PubsubClient<
     request.setResumeAtTopicSequenceNumber(
       options.subscriptionState.resumeAtTopicSequenceNumber
     );
+    request.setSequencePage(
+      options.subscriptionState.resumeAtTopicSequencePage
+    );
+
+    this.getLogger().trace(
+      `Subscribing to topic with resume_at_topic_sequence_number ${options.subscriptionState.resumeAtTopicSequenceNumber} and sequence_page ${options.subscriptionState.resumeAtTopicSequencePage}`
+    );
```

Comment:
while testing locally, I noticed the logs coming from the web sdk looked poorly formatted, reverted to using the string interpolation instead

```
[2024-11-08T18:08:38.251Z] TRACE (Momento: PubsubClient): Issuing publish request; topic: %s, message length: %stopic-web,15
[2024-11-08T18:08:38.436Z] TRACE (Momento: PubsubClient): Received an item on subscription stream; topic: %s; sequence number: %s; sequence page: %stopic-web,3,1731089315
```